pkgs/by-name/cp/cpu-energy-meter/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libcap
+}:
+stdenv.mkDerivation rec {
+  pname = "cpu-energy-meter";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "sosy-lab";
+    repo = "cpu-energy-meter";
+    rev = version;
+    hash = "sha256-QW65Z8mRYLHcyLeOtNAHjwPNWAUP214wqIYclK+whFw=";
+  };
+
+  patchPhase = ''
+    substituteInPlace Makefile \
+      --replace "DESTDIR :=" "DESTDIR := $out" \
+      --replace "PREFIX := /usr/local" "PREFIX :="
+  '';
```

[Don't forget the `runHook`s :) (see checklist)](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#new-packages)

```suggestion
  patchPhase = ''
    runHook prePatch
  
    substituteInPlace Makefile \
      --replace "DESTDIR :=" "DESTDIR := $out" \
      --replace "PREFIX := /usr/local" "PREFIX :="
      
    runHook postPatch
  '';
```